tests/versions/apm_server.yml

```diff
@@ -1,4 +1,12 @@
 APM_SERVER:
-  - master
-  - master --oss
-  - master --ubi8
+  - '7.x'
+  - '7.10'
+  - '7.9;--release'
+  - '7.8;--release'
+  - '7.7;--release'
+  - '7.6;--release'
+  - '7.5;--release'
+  - '7.4.2;--release'
```

Comment:
For some reason it tries to use 7.4.3 instead 7.4.2


```
 

[2020-11-16T13:37:28.661Z] ERROR: for kibana  manifest for docker.elastic.co/kibana/kibana:7.4.3 not found: manifest unknown: manifest unknown

[2020-11-16T13:37:28.661Z] manifest for docker.elastic.co/apm/apm-server:7.4.3 not found: manifest unknown: manifest unknown

[2020-11-16T13:37:28.661Z] manifest for docker.elastic.co/elasticsearch/elasticsearch:7.4.3 not found: manifest unknown: manifest unknown

[2020-11-16T13:37:28.661Z] manifest for docker.elastic.co/kibana/kibana:7.4.3 not found: manifest unknown: manifest unknown
```

As far as I see the 7.x branch use 7.4.2 but master uses 7.4.3

Reply:
See https://github.com/elastic/apm-integration-testing/pull/982